workalendar/core.py

```diff
@@ -982,7 +982,7 @@ def export_to_ical(self, period=[2000, 2030], target_path=None):
                 f'SUMMARY:{name}',
                 f'DTSTART;VALUE=DATE:{date_:%Y%m%d}',
                 dtstamp,
-                f'UID:{date_}{name}@peopledoc.github.io/workalendar',
+                f'UID:{date_}{name}@workalendar.github.io/workalendar',
```

Comment:
We can revert this one if you think that could have an impact on existing ical users.

Reply:
well, not sure. Let's say we have the following use-case:

* someone has built a calendar for their country yesterday (before this patch), using a period of time, let's say [2021, 2022].
* time passes. this patch is merged, released and published
* the same person now generates a new file, probably using a new period

If they import this ical file in their favorite calendar app, what happens?

Reply:
I'm guessing they'll have duplicate events. That's why I suggest maybe this one cas stay as-is.